---
title: "Final Report: Your Title Here"
author: "Don Francisco"
date: "November 22, 2017"
output:
    github_document: default
    word_document: default
bibliography: references.bib
csl: bioinformatics.csl
---

# Introduction

Add about 2-3 pages here. Across the whole manuscript, you should cite at least 20 peer reviewed articles.

# Methods

## Study design

The original study by Jangi *et al.* sampled the gut microbiome of 103 individuals, 60 multiple sclerosis (MS) patients and 43 healthy controls. [@jangi2016alterations] there were three variations of MS patients: those with untreated MS, those recieving treatment from beta-interferon, and those recieving treatment with glatiramer acetate. Both groups that had been recieving tratment had been doing so for at least 6 months. Both MS and control patients had no antibiotic use in the last 6 months, no probiotic use,  no corticosteroids, np history of gastroenteritis, no irritable bowel syndrome, no bowel surgery, no bowel disease, no other autoimmune disease, were not pregnant, and had not traveled outside the country in the past month. All patients were also given a dietary survey before the collection of samples. 

## Sample origin and sequencing

To obtain microbiome data, Jangi *et al.* used stool samples that were collected by the patients and shipped to the laboratory overnight a 0 degrees Celcius. [@jangi2016alterations] Upon recieving the smples, they were stored at -80 degrees Celcius until DNA extaction, and were only subjected to one freeze-thaw cycle. They obtained sequence data from the fecal microbes using both pyrosequencing and Illumina sequencing of the 16S rRNA gene. 

## Computational

I processed and analyzed the sequence data using a combination of bash and R. First, I used bash to download and process the samples in order to determine their length and quality. For this analysis, I only used the pyrosequencing-derived sequences from the original study. Sequences of sufficient quality were then analyzed in R, using all of the relevant metadata and sequence information. R analyses were conducted through the usage of base R, as well as DADA2 and phyloseq. I made some additional analysis by exporting the fasta files derived from DADA2 into Geneious. [@geneious] There, I used the FastTree plugin to constrict the phylogenetic tree that went into the phyloseq object. [@fasttree] Phyloseq was used to add various microbiome-specific functions to ggplot in order to better visualize the data [@mcmurdie2013].

DADA2 was used for inter-sequence analysis and cleaning, such as forming OTUs and resolving minor sequence differences. The DADA2 pipeline begins with filtering paired fastq files by trimming them to a specific length and removing sequences that are too short, then further filtering based on number of expected errors, quality score, and number of ambiguous bases [@callahan2016]. Next, it can remove chimeras, which are sequences that are mare up of two parent sequences that result from sequencing errors. Lastly, it merges paired forward and reverse reads after all the sequences have been cleaned and de-noised, so that DADA2 can be extremely strict and require exact overlap.

# Results

```{r load-libraries}
# load all necesary libraries
library("dplyr")
library("tidyr")
library("knitr")
library("ggplot2")
library("vegan")
library("RColorBrewer")
library("citr")
library("seqinr")
library("mctoolsr")
library("phyloseq")

# load the output of the DADA2 script
load("output/phyloseq_obj.RData")
seq_table <- read.table("output/sequence_variants_table.txt",
                        row.names = 1,
                        header = TRUE)
metadata_in <- read.table(paste0("data/metadata/",
                                 "SraRunTable.txt"),
                          sep = "\t",
                          header = TRUE,
                          stringsAsFactors = FALSE,
                          row.names = 13)
# Melt phyloseq object to make certain analysis easier
melted_obj <- psmelt(phyloseq_obj)
```

##Ovelall Community Analysis

```{r overall-diversity}
# this subset removes mouse data from the dataset
plot_richness(phyloseq_obj,
              x = "host_phenotype_s",
              measures = c("Shannon")) +
  xlab("Host Disease Status") +
  geom_boxplot(aes(fill = host_phenotype_s),
               width = 0.2) +
  ggtitle("Alpha diversity in MS versus healthy patients")
```
**Figure 1.** Alpha diversity in patients by disease status
This figure shows Shannon diversity in healthy control patients and MS patients receiving each type of treatment.

**Table 1.** Phylum abindance in each 
This table shows the abundance of each phylum in healthy patients and MS patients of each treatment type.
```{r phylum-by-phenotype}
# this code plots the abundance of each phylum present in each sex's hands
phylum_totals <- melted_obj %>%
  filter(!is.na(Phylum)) %>%
  group_by(Phylum, host_phenotype_s) %>%
  tally() %>%
  spread(key = host_phenotype_s,
         value = n) %>%
  arrange(desc(Healthy_Control), desc(Multiple_Sclerosis_Interferon),
          desc(Multiple_Sclerosis_Copaxone), desc(Multiple_Sclerosis_Untreated))
kable(phylum_totals)
```

**Table 2.** Number of patients of each treatment type
This table shows the number of patients in each treatment group. Each MS group is smaller on its own than the control group, and the different treatment groups are smaller than the untreated group.
```{r patient-numbers}

melted_obj %>%
  group_by(host_phenotype_s) %>%
  summarise(number_of_patients = n_distinct(isolate_s))
```

```{r phylum_graph}
# this makes a stacked bar plot with phylum relative abundances
# for each type of patient
melted_obj %>%
  filter(!is.na(Phylum)) %>%
  group_by(host_phenotype_s, Phylum) %>%
  summarize(mean_abund = mean(Abundance)) %>%
  ggplot(aes(x = host_phenotype_s,
             y = mean_abund,
             fill = Phylum)) +
    geom_col(position = "fill",
             color = "black") +
  #scale_x_discrete(labels = c("a", "b", "c", "d")) +
  theme(axis.text.x = element_text(angle = 45, hjust = 1)) +
  ggtitle("A")

melted_obj %>%
  filter(!is.na(Phylum)) %>%
  group_by(host_phenotype_s, Phylum) %>%
  summarize(sd_abund = sd(Abundance)) %>%
  ggplot(aes(x = host_phenotype_s,
             y = sd_abund,
             fill = Phylum)) +
    geom_col(position = "fill",
             color = "black") +
  #scale_x_discrete(labels = c("a", "b", "c", "d")) +
  theme(axis.text.x = element_text(angle = 45, hjust = 1)) +
  ggtitle("B")
```
**Figure 2.** Normalized abundances of each phylum
This figure shows the normalized abundances of each phylum in each treatment group (A), and their respective standard deviations (B)

```{r phylogeny}
#this code displays the phylogenetic tree from 
plot_tree(phyloseq_obj,
          color = "host_phenotype_s",
          ladderize = TRUE) +
# the last line arranges the tree branches from short to long
  ggtitle("A. Phylogenetic tree colored by host ID")
# you can use prune_taxa to look at trees for certain taxa
```
**Figure 3.** Phylogenetic tree colored by patient disease status
This figure shows a phylogenetic tree of all sequences in the dataset, with tips colored by patient disease status.

##Firmicutes Phylum Analysis

**Table 3.** Common classes, orders, and families in Firmicutes bacteria
This table shows the abundances of each class, order, and family within the Firmicutes phylum for MS patient sof all disease statuses.
```{r fir-common-classes}
firm_classes <- melted_obj %>%
  filter(Phylum == "Firmicutes") %>%
  filter(!is.na(Class)) %>%
  group_by(host_phenotype_s, Class, Order, Family) %>%
  tally() %>%
  spread(key = host_phenotype_s,
         value = n) %>%
  arrange(desc(Healthy_Control), desc(Multiple_Sclerosis_Interferon),
          desc(Multiple_Sclerosis_Copaxone), desc(Multiple_Sclerosis_Untreated))
kable(firm_classes)
```

```{r fir-diversity}

fir_subset <- subset_taxa(phyloseq_obj, Phylum == "Firmicutes")
# this subset removes mouse data from the dataset
plot_richness(fir_subset,
              x = "host_phenotype_s",
              measures = c("Shannon")) +
  xlab("Host Disease Status") +
  geom_boxplot(aes(fill = host_phenotype_s),
               width = 0.2) +
  ggtitle("Alpha diversity  of Firmicutes in MS versus healthy patients")
```

```{r phylogeny_fir}

plot_tree(fir_subset,
     color = "host_phenotype_s")

```

##Cyanobacteria Community Analysis

```{r cy-common-classes}
cy_class <- melted_obj %>%
  filter(Phylum == "Cyanobacteria/Chloroplast") %>%
  group_by(host_phenotype_s, Class) %>%
  tally() %>%
  spread(key = host_phenotype_s,
         value = n) %>%
  arrange(desc(Healthy_Control), desc(Multiple_Sclerosis_Interferon),
          desc(Multiple_Sclerosis_Copaxone), desc(Multiple_Sclerosis_Untreated))
kable(cy_class)

```

```{r cy-diversity}

cy_subset <- subset_taxa(phyloseq_obj, Phylum == "Cyanobacteria/Chloroplast")
# this subset removes mouse data from the dataset
plot_richness(cy_subset,
              x = "host_phenotype_s",
              measures = c("Shannon")) +
  xlab("Host Disease Status") +
  geom_boxplot(aes(fill = host_phenotype_s),
               width = 0.2) +
  ggtitle("Alpha diversity of Cyanobacteria in MS versus healthy patients")
```

```{r phylogeny_cy}

plot_tree(cy_subset,
     color = "host_phenotype_s")

```

##Verrucomicrobia Phylum Analysis

```{r ver-common-classes}
ver_classes <- melted_obj %>%
  filter(Phylum == "Verrucomicrobia") %>%
  group_by(host_phenotype_s, Class, Order, Family, Genus) %>%
  tally() %>%
  spread(key = host_phenotype_s,
         value = n) %>%
  arrange(desc(Healthy_Control), desc(Multiple_Sclerosis_Interferon),
          desc(Multiple_Sclerosis_Copaxone), desc(Multiple_Sclerosis_Untreated))
kable(ver_classes)

```

```{r cy-phylogeny}

ver_subset <- subset_taxa(phyloseq_obj, Phylum == "Verrucomicrobia")
# this subset removes mouse data from the dataset
plot_richness(ver_subset,
              x = "host_phenotype_s",
              measures = c("Shannon")) +
  xlab("Host Disease Status") +
  geom_boxplot(aes(fill = host_phenotype_s),
               width = 0.2) +
  ggtitle("Alpha diversity of Verrucomicrobia in MS versus healthy patients")
```

```{r phylogeny_ver}

plot_tree(ver_subset,
     color = "host_phenotype_s")

```
In addition to a minimum of 5-10 figures/tables (and associated captions), you should include sufficient text in this section to describe what your findings were. Remember that in the results section you just describe what you found, but you don't interpret it - that happens in the discussion. 2-3 pages.

# Discussion

Add around 3-4 pages interpreting your results and considering future directions one might take in analyzing these data.

# Sources Cited
