---
title: "Final Report: The Human Gut Microbiome in Multiple Sclerosis Patients Recieving Various Treatments"
author: "Emma Gibson"
date: "December 14, 2017"
output:
    word_document: default
    github_document: default
bibliography: references.bib
csl: bioinformatics.csl
---

# Introduction

*Background.* In recent years, the human microbiome and its role and function in human health has become a popular topic of study and research. Unlike the harmful pathogens that bacteria are commonly assumed to be, these microbes live symbiotically on and within our body. For instance, studies have shown that they play a critical role in digestion for numerous eukaryotic organisms, from humans to abalone. [@ley2008evolution; @erasmus1997role] Although these microbial communities can have vast and numerous affects on their hosts, it was incredibly difficult to study them until recent breakthroughs in DNA sequencing due to their microscopic size and incredible abundance. Now that we have the means to study them, scientists are discovering that their effects on host physiology are numerous and profound.

Although the microbes inside our gut are primarily beneficial in nature, these communities have also been linked to countless ailments and diseases, including many autoimmune disorders. [@clemente2012impact] While their purpose is complex and still not entirely understood, gut microbes are known to play a role in training the immune system to recognize and attack potential pathogens both within the gut and throughout the rest of the body. [@mcdermott2014microbiome; @lee2010has] In theory, this means that an abnormal microbiome could potentially lead to a poorly-trained immune system and thereby contribute to autoimmune disorders. Therefore, there is plentiful research being conducted on the role that these microbial communities both inside the digestive tract and elsewhere in the body can play immune system-related disorders.

The disorders that are linked tot he microbiome are thought to not be caused by some kind of pathogenesis, but instead by *dysbiosis*. A state of *dysbiosis*, or an abnormal gut microbiome, has already been correlated with numerous disorders, including certain types of diabetes, irritable bowel disorder, and rheumatoid arthritis. [@kostic2014microbiome; @alkanani2015alterations; @scher2013expansion] In this analysis, I chose to focus on the link between the gut microbiome and multiple sclerosis, another autoimmune disorder that has been linked to dysbiosis. [@cantarel2015gut; @chen2016multiple] 

*Study system.* Multiple sclerosis is an autoimmune disease characterized by the immune system's T-cells attacking the myelin coating of nerve cells, causing nerve damage. [@miranda2013role] Symptoms can include pain, motor control issues, sensory issues, and mood issues. Currently, it is unclear what causes the disorder, and studies have found numerous possible causes and risk factors. Some studies have linked its development to environmental factors, such as sunlight exposure and diet. [@riccio2010may] There have also been several possible genetic markers found throughout the human genome. [@oksenberg2008genetics] Although multiple sclerosis is likely the result of numerous interacting environmental and genetic factors, the gut microbiome can act as an interface for all of these factors, leading to the development of the disease. [@ochoa2014gut; @WANG20141]

Like the previously mentioned autoimmune disorders, multiple sclerosis has been correlated with a state of *dysbiosis*. [@miyake2015dysbiosis] In evaluating the role that the gut microbiome plays in the development and treatment of such diseases, it is important to note that correlation does not necessarily equal causation. It could be possible that *dysbiosis* is a result of multiple sclerosis, rather than a cause. However, a recent study by Berer *et al.* has demonstrated that gut microbes from multiple sclerosis patients can induce multiple sclerosis-like symptoms in transgenic mouse models. [@berer2017gut] This indicates that the gut microbiome is not simply an indicator of multiple sclerosis, but plays a role it the development of the disorder as well.

*Objectives of this study.* In this analysis, I aim to discover the primary differences between gut microbiomes of multiple sclerosis patients and healthy patients, as well as the differences between different multiple sclerosis treatments. One aspect of this analysis will involve low-resolution analysis of the microbiome community as a whole. I looked at the relative abundances of various bacterial phyla among the different patients, looking for differences in the ratios of one bacterial clade to the other. I expected to find differences in the overall diversity between patients of different disease types, with multiple sclerosis patients having less overall diversity. I expected the multiple sclerosis patients to have less diversity because I thought that a state of dysbiosis would lead to an overall less populous gut microbiome. I also expected significant differences in the abundances of certain phyla between healthy controls and multiple sclerosis patients.Using similar logic, I predicted that the dysbiotic state of the multiple sclerosis microbiome would likely show greater abundances of certain bacteria, and lower abundances of others.

I also conducted more detailed analysis at several bacterial phyla too see if there are notable trends at the species, genus, or other higher-resolution levels. At this more detailed level, I expected to find phylogenetic differences between bacteria found in multiple sclerosis patients versus healthy controls. I figured that if there was a difference between the microbiomes of these two groups, then there should be distinct phylogenetic clades that contain either multiple sclerosis patients' or healthy controls' bacteria. The main purpose of this study was to discover what trends there are among multiple sclerosis patient and healthy control microbiomes, in both general and detailed terms.

# Methods

## Study design

The original study by Jangi *et al.* sampled the gut microbiome of 103 individuals, 60 multiple sclerosis patients and 43 healthy controls. [@jangi2016alterations] There were three variations of multiple sclerosis patients: those with untreated multiple sclerosis, those receiving treatment from beta-interferon (Interferon), and those receiving treatment with glatiramer acetate (Copaxone). Both groups that had been receiving treatment had been doing so for at least 6 months. Both multiple sclerosis and control patients had no antibiotic use in the last 6 months, no probiotic use,  no corticosteroids, no history of gastroenteritis, no irritable bowel syndrome, no bowel surgery, no bowel disease, no other autoimmune disease, were not pregnant, and had not traveled outside the country in the past month. All patients were also given a dietary survey before the collection of samples. 

## Sample origin and sequencing

To obtain microbiome data, Jangi *et al.* used stool samples that were collected by the patients and shipped to the laboratory overnight a 0 degrees Celsius. [@jangi2016alterations] Upon receiving the samples, they were stored at -80 degrees Celsius until DNA extraction, and were only subjected to one freeze-thaw cycle. They obtained sequence data from the fecal microbes using both Roche 454 pyrosequencing and Illumina MiSeq sequencing of the 16S rRNA gene. 

## Computational

I processed and analyzed the sequence data using a combination of bash and R. First, I used bash to download and process the samples in order to determine their length and quality. For this analysis, I only used the pyrosequencing-derived sequences from the original study, because attempting to combine pyrosequencing and Illumina sequencing results would prove troublesome. Before cleaning the sequences, I reformatted some of the metadata, because some of the labels pertaining to patient disease status were inconsistent or contained typos. Although it was still simple enough for a human to understand what the labels meant, they contained minor errors such as spaces and inconsistent capitalization, which would cause trouble for R analysis if not corrected. After correcting these minor metadata errors, I moved on to cleaning and trimming the sequences.

DADA2 was used for inter-sequence analysis and cleaning, such as forming OTUs and resolving minor sequence differences. The DADA2 pipeline began with filtering paired fastq files by trimming them to a specific length and removing sequences that were too short, then further filtering based on number of expected errors, quality score, and number of ambiguous bases [@callahan2016]. Because I was working with pyrosequencing data that was quite lengthy, I set my maximum sequence length to 700 base pairs. Next, DADA2 removed the chimeras, which are sequences that are mare up of two parent sequences that result from sequencing errors.

Sequences of sufficient quality were then analyzed in R, using all of the relevant metadata and sequence information. R analyses were conducted using base R, as well as ggplot, Geneious, and phyloseq. I made some additional analysis by exporting the fasta files derived from DADA2 into Geneious. [@geneious] There, I used the FastTree plugin to construct the phylogenetic tree that went into the phyloseq object. [@fasttree] Phyloseq was used to add various microbiome-specific functions to ggplot in order to better visualize the data, such as assigning taxonomies and allowing phylogenetic tree manipulation. [@mcmurdie2013]

# Results

```{r load-libraries, echo=FALSE, message=FALSE}
# load all necesary libraries
library("dplyr")
library("tidyr")
library("knitr")
library("ggplot2")
library("vegan")
library("RColorBrewer")
library("citr")
library("seqinr")
library("mctoolsr")
library("phyloseq")

# load the output of the DADA2 script
load("output/phyloseq_obj.RData")
seq_table <- read.table("output/sequence_variants_table.txt",
                        row.names = 1,
                        header = TRUE)
metadata_in <- read.table(paste0("data/metadata/",
                                 "SraRunTable.txt"),
                          sep = "\t",
                          header = TRUE,
                          stringsAsFactors = FALSE,
                          row.names = 13)
# Melt phyloseq object to make certain analysis easier
melted_obj <- psmelt(phyloseq_obj)
```

##Overall Community Analysis

```{r overall-diversity, echo=FALSE, message=FALSE, warning=FALSE, fig.width=7}
# this finds Shannon diversity in all patients
plot_richness(phyloseq_obj,
              x = "host_phenotype_s",
              measures = c("Shannon")) +
  xlab("Host Disease Status") +
  geom_boxplot(aes(fill = host_phenotype_s),
               width = 0.2) +
    scale_x_discrete(labels = c("Healthy",
                              "MS (Copaxone)",
                              "MS (Interferon)",
                              "MS (Untreated))")) +
  theme(axis.text.x = element_text(angle = 0, hjust = 1)) +
  ggtitle("Alpha diversity in MS versus healthy patients")
```

**Figure 1.** Alpha diversity in patients by disease status

This figure shows Shannon diversity in healthy control patients and multiple sclerosis patients receiving each type of treatment. Diversity is relatively similar in all samples regardless of multiple sclerosis status or treatment status. However, the multiple sclerosis groups consistently show a slight increase in diversity, especially in the Copaxone group.

**Table 1.** Phylum abundance in each 

This table shows the abundance of each phylum in healthy patients and multiple sclerosis patients of each treatment type. Although the numbers are hard to compare between groups because the patient groups are of different sizes, there are consistencies in which bacterial phyla are the most and least common. 

```{r phylum-by-phenotype, echo=FALSE}
# this code gets the abundance of each phylum present in each patient type
phylum_totals <- melted_obj %>%
  filter(!is.na(Phylum)) %>%
  group_by(Phylum, host_phenotype_s) %>%
  tally() %>%
  spread(key = host_phenotype_s,
         value = n) %>%
  arrange(desc(Healthy_Control), desc(Multiple_Sclerosis_Interferon),
          desc(Multiple_Sclerosis_Copaxone), desc(Multiple_Sclerosis_Untreated))
kable(phylum_totals)
```

**Table 2.** Number of patients of each treatment type

This table shows the number of patients in each treatment group. Each multiple sclerosis group is smaller on its own than the control group, and the different treatment multiple sclerosis groups are smaller than the untreated multiple sclerosis group. The differences in sample sizes roughly correlate with the total abundances of each phylum. (Table 1) For instance, the ratio of total abundance of Tenericutes among different patient groups is almost identical to the ratio of patient numbers in each group. These discrepancies in sample sized make it necessary to normalize data when taking measurements like sum abundance into account.

```{r patient-numbers, echo=FALSE}
# this code tells the number of patients with each disease type
melted_obj %>%
  group_by(host_phenotype_s) %>%
  summarise(number_of_patients = n_distinct(isolate_s)) %>%
  kable()
```

```{r phylum_graph, echo=FALSE, fig.width=7}
# this makes a stacked bar plot with phylum relative abundances
# for each type of patient
melted_obj %>%
  filter(!is.na(Phylum)) %>%
  group_by(host_phenotype_s, Phylum) %>%
  summarize(mean_abund = mean(Abundance)) %>%
  ggplot(aes(x = host_phenotype_s,
             y = mean_abund,
             fill = Phylum)) +
    geom_col(position = "fill",
             color = "black") +
  scale_x_discrete(labels = c("Healthy",
                              "MS (Copaxone)",
                              "MS (Interferon)",
                              "MS (Untreated))")) +
  theme(axis.text.x = element_text(angle = 0, hjust = 1)) +
  ggtitle("(A) Phylum abundance in MS vs healthy patients")
# this performs a similar analysis, but with standard deviation
melted_obj %>%
  filter(!is.na(Phylum)) %>%
  group_by(host_phenotype_s, Phylum) %>%
  summarize(sd_abund = sd(Abundance),
            mean_abund = mean(Abundance)) %>%
  ggplot(aes(x = host_phenotype_s,
             color = Phylum)) +
    geom_errorbar(aes(ymin = mean_abund - sd_abund,
                      ymax = mean_abund + sd_abund),
                  width = 0.25) +
  geom_point(aes(y = mean_abund)) +
  facet_wrap(~ Phylum) +
  scale_x_discrete(labels = c("Healthy",
                              "MS (Copaxone)",
                              "MS (Interferon)",
                              "MS (Untreated)")) +
  theme(axis.text.x = element_text(angle = 50, hjust = 1)) +
  ggtitle("(B) Phylum abundance standard deviation
          in MS vs healthy patients")
```

**Figure 2.** Normalized abundances and standard deviations of each phylum for patients of all types

This figure shows the normalized abundances of each phylum in each treatment group, and their respective standard deviations. (A) shows the normalized abundances of each phylum because the overall sum data is heavily skewed due to diverse sample sized for each test group, which are detailed in Table 2. Although there are several groups of bacteria that retain consistent relative abundance across all patient types, the Interferon-treated multiple sclerosis shows some major differences. Most notably, it contains considerably more Cyanobacteria and less Verrucomicrobia than the other patient groups. (B) shows the normalized standard deviations of each phylum, which remain fairly consistent across patient types. However, once again the Interferon group shows an increased standard deviation among Cyanobacteria and a decreased standard deviation among Verrucomicrobia.

```{r phylogeny, echo=FALSE, fig.width=7}
# this code displays the phylogenetic tree from
# the GENEious analysis
plot_tree(phyloseq_obj,
          color = "host_phenotype_s",
          ladderize = TRUE) +
# the last line arranges the tree branches from short to long
  ggtitle("Phylogenetic tree colored by host disease type")
# you can use prune_taxa to look at trees for certain taxa
```

**Figure 3.** Phylogenetic tree colored by patient disease status

This figure shows a phylogenetic tree of all sequences in the data set, with tips colored by patient disease status.Although there are many clear and well-supported clades, the large number of diverse sequences make the tree quite large. At such a broad scope, there are no clear patterns or clades that correlate to patients of one treatment status or another.

##Firmicutes Phylum Analysis

**Table 3.** Common classes in Firmicutes bacteria

This table shows the abundances of each class and order within the Firmicutes phylum for multiple sclerosis patients of all disease statuses. Samples whose class could not be determined were excluded. Being one of the most abundant phyla in all samples, there is a large number of classes and orders present in patients of all types. Despite this wide variety of classes, just like with the phylum abundance metrics, the actual sum abundance of each patient type is skewed by its respective sample size.

```{r fir-common-classes, echo=FALSE}
# this subsets the bacteria in the Firmicutes phylum
# and finds the subsequent classes
firm_classes <- melted_obj %>%
  filter(Phylum == "Firmicutes") %>%
  filter(!is.na(Class)) %>%
  group_by(host_phenotype_s, Class, Order) %>%
  tally() %>%
  spread(key = host_phenotype_s,
         value = n) %>%
  arrange(desc(Healthy_Control), desc(Multiple_Sclerosis_Interferon),
          desc(Multiple_Sclerosis_Copaxone), desc(Multiple_Sclerosis_Untreated))
# this prints the table in a markdown-friendly format
kable(firm_classes)
```

```{r fir-diversity, echo=FALSE, message=FALSE, warning=FALSE, fig.width=7}
# This subsets the data to only include Firmicutes phylum
fir_subset <- subset_taxa(phyloseq_obj, Phylum == "Firmicutes")
# This finds the shannon diversity within this subset
plot_richness(fir_subset,
              x = "host_phenotype_s",
              measures = c("Shannon")) +
  xlab("Host Disease Status") +
  geom_boxplot(aes(fill = host_phenotype_s),
               width = 0.2) +
  scale_x_discrete(labels = c("Healthy",
                              "MS (Copaxone)",
                              "MS (Interferon)",
                              "MS (Untreated))")) +
  theme(axis.text.x = element_text(angle = 0, hjust = 1)) +
  ggtitle("Alpha diversity  of Firmicutes in MS versus healthy patients")
```

**Figure 4.** Shannon diversity in Firmicutes bacteria

This figure shows the Shannon diversity in all bacteria within the Firmicutes phylum for patients of all disease statuses. The diversity patterns within this phylum more or less reflect the diversity of the overall data. That said, there appears to be a slightly higher amount of variability within the diversity for each patient type within this phylum than in the overall community data.

```{r phylogeny_fir, echo=FALSE, fig.width=7}
# this makes a phylogenetic tree of the Firmicutes subset
plot_tree(fir_subset,
     color = "host_phenotype_s") +
  ggtitle("Phylogenetic tree colored by host disease type")
```

**Figure 5.** Phylogenetic tree of Firmicutes bacteria

This figure shows a phylogenetic tree of all sequences in the Firmicutes phylum, with tips colored by patient disease status. While there are several prominent clades, there are no clear patterns where one clade contains predominantly patients of a certain treatment or disease status. As with the overall community analysis, this phylum appears to contain too many diverse sequences to construct a detailed phylogenetic tree.

```{r fir_order_abindance, echo=FALSE}
# this makes a stacked bar plot with relative abundances
# of each order in Firmicutes for each type of patient
# first, I made a subset with all Firmicutes bacteria
fir_ord_ab <- melted_obj %>%
  filter(Phylum == "Firmicutes") %>%
  filter(!is.na(Class)) %>%
  group_by(host_phenotype_s, Class, Order)
# then, I find and graph mean adundances
fir_ord_ab %>%
  summarize(mean_abund = mean(Abundance)) %>%
  ggplot(aes(x = host_phenotype_s,
             y = mean_abund,
             fill = Order)) +
    geom_col(position = "fill",
             color = "black") +
  scale_x_discrete(labels = c("Healthy",
                              "MS (Copaxone)",
                              "MS (Interferon)",
                              "MS (Untreated)")) +
  theme(axis.text.x = element_text(angle = 0, hjust = 1)) +
  ggtitle("Class abundance in Firmicutes in MS vs healthy patients")
```

**Figure 10.** Normalized abundances of each order within Firmicutes for patients of each treatment type

This figure shows the normalized relative abundances of each order within Firmicutes. Most notable trends include the reduction of Bacillales in Copaxone-treated and untreated multiple sclerosis patients, a decrease in Lactobacillales in Copaxone patients, and an increase in Lactobacillales in Interferon-treated MS patients.

##Cyanobacteria Community Analysis

**Table 4.** Common class in Cyanobacteria

This table shows the class of Cyanobacteria samples for multiple sclerosis patients of all disease statuses. Samples whose class could not be determined were not included. All samples were identified to be from the same class, Chloroplast, across both healthy controls and patients of all disease statuses.

```{r cy-common-classes, echo=FALSE}
# this subsets the bacteria in the Cyanobacteria phylum
# and identifies the sunsequent classes
cy_class <- melted_obj %>%
  filter(Phylum == "Cyanobacteria/Chloroplast") %>%
  group_by(host_phenotype_s, Class) %>%
  tally() %>%
  spread(key = host_phenotype_s,
         value = n) %>%
  arrange(desc(Healthy_Control), desc(Multiple_Sclerosis_Interferon),
          desc(Multiple_Sclerosis_Copaxone), desc(Multiple_Sclerosis_Untreated))
# this prints the above table in a markdown-friendly format
kable(cy_class)
```

```{r cy-diversity, echo=FALSE, message=FALSE, warning=FALSE, fig.width=7}
# This subsets the bacteria in the Cyanobacteria phylum
cy_subset <- subset_taxa(phyloseq_obj, Phylum == "Cyanobacteria/Chloroplast")
# this plots the shannon diversity in said subset
plot_richness(cy_subset,
              x = "host_phenotype_s",
              measures = c("Shannon")) +
  xlab("Host Disease Status") +
  geom_boxplot(aes(fill = host_phenotype_s),
               width = 0.2) +
  scale_x_discrete(labels = c("Healthy",
                              "MS (Copaxone)",
                              "MS (Interferon)",
                              "MS (Untreated))")) +
  theme(axis.text.x = element_text(angle = 0, hjust = 1)) +
  ggtitle("Alpha diversity of Cyanobacteria in MS versus healthy patients")
```

**Figure 6.** Shannon diversity in Cyanobacteria

This figure shows the Shannon diversity in all bacteria within the Cyanobacteria phylum for patients of all disease statuses. There is almost no diversity in any of the samples, which is reflected in Table 4, that shows there is only one class within this phylum. There is one outlier, which is within the patients being treated with Interferon. This is the same disease status group that showed an increase in Cyanobacteria (Figure 2)

```{r phylogeny_cy, echo=FALSE, fig.width=7}
# This makes a phylogenetic tree of Cyanobacteria
plot_tree(cy_subset,
     color = "host_phenotype_s") +
  ggtitle("Phylogenetic tree colored by host disease type")
```

**Figure 7.** Phylogenetic tree of Cyanobacteria

This figure shows a phylogenetic tree of all sequences in the Cyanobacteria phylum, with tips colored by patient disease status. There are two major well-supported clades, one that contains primarily Cyanobacteria from healthy patients, and one that contains exclusively multiple sclerosis patients.

##Verrucomicrobia Phylum Analysis

**Table 5.** Common class, order, family, and genus in Verrucomicrobia bacteria

This table shows the class, order, family, and genus among Verrucomicrobia samples for multiple sclerosis patients of all disease statuses. Samples whose class could not be determined were not included. All samples were identified to be from the same class, order, family, and genus across both healthy controls and patients of all disease statuses.

```{r ver-common-classes, echo=FALSE}
# this subsets the bacteria in the Verrucomicrobia phylum
# and identifies the sunsequent classes
ver_classes <- melted_obj %>%
  filter(Phylum == "Verrucomicrobia") %>%
  group_by(host_phenotype_s, Class, Order, Family, Genus) %>%
  tally() %>%
  spread(key = host_phenotype_s,
         value = n) %>%
  arrange(desc(Healthy_Control),
          desc(Multiple_Sclerosis_Interferon),
          desc(Multiple_Sclerosis_Copaxone),
          desc(Multiple_Sclerosis_Untreated))
# this puts the above information in a markdown-friendly format
kable(ver_classes)
```

```{r ver-diversity, echo=FALSE, message=FALSE, warning=FALSE, fig.width=7}
# This subsets the bacteria in the Verrucomicrobia phylum
ver_subset <- subset_taxa(phyloseq_obj, Phylum == "Verrucomicrobia")
# this plots the shannon diversity of said subset
plot_richness(ver_subset,
              x = "host_phenotype_s",
              measures = c("Shannon")) +
  xlab("Host Disease Status") +
  geom_boxplot(aes(fill = host_phenotype_s),
               width = 0.2) +
  scale_x_discrete(labels = c("Healthy",
                              "MS (Copaxone)",
                              "MS (Interferon)",
                              "MS (Untreated))")) +
  theme(axis.text.x = element_text(angle = 0, hjust = 1)) +
  ggtitle("Alpha diversity of Verrucomicrobia in MS versus healthy patients")
```

**Figure 8.** Shannon diversity in Verrucomicrobia bacteria

This figure shows the Shannon diversity in all bacteria within the Verrucomicrobia phylum for patients of all disease statuses. There is almost no diversity in any of the samples, which is reflected in Table 5, that shows all of the bacteria in this phylum belongs to the same class, order, family, and genus.

```{r phylogeny_ver, echo=FALSE, fig.width=7}
# this makes a phylogenetic tree of bacteria in Verrucomicrobia
plot_tree(ver_subset,
     color = "host_phenotype_s") +
  ggtitle("Phylogenetic tree colored by host disease type")
```

**Figure 9.** Phylogenetic tree of Verrucomicrobia

This figure shows a phylogenetic tree of all sequences in the Verrucomicrobia phylum, with tips colored by patient disease status. There are no obvious clades within this tree, dominated by one patient types or otherwise. In addition, all of the tips are very short, indicating that all of the sequences in this data set are fairly similar and  reinforcing the fact that there is little diversity within this phylum. (Figure 8)

# Discussion

##Overall Community Analysis

The differences in overall diversity between multiple sclerosis patients and the healthy controls were somewhat negligible, although all multiple sclerosis groups showed slightly higher diversity than the control patients. (Figure 1) This refutes my initial hypothesis that multiple sclerosis patients would have less overall diversity, because it appears that they actually have similar if not slightly higher diversity than the healthy controls. The reason for this increased diversity is unclear based on my analysis, so further study would be needed to investigate this completely. One possible explanation could be that multiple sclerosis is cause by the presence of harmful bacteria, rather than the absence of beneficial ones. However, the correlation is not strong enough to draw any significant conclusions from.

When looking at the overall abundance of each phylum in the patients, it is important to remember that the various sample sizes will influence the overall sum abundance. (Table 2) Keeping hat in mind, it is still apparent that the most common bacteria in multiple sclerosis patients are the same as those in healthy patients, such as the Firmicutes and Bacteriodetes phyla. Given that these are generally considered the most common phyla in the human gut, this is not surprising. [@turnbaugh2009core]

That said, there did appear to be significant differences in community composition between the control patients, as well as between patients receiving different treatment types. (Figure 2 A) Most notably, patients receiving Interferon treatment for their multiple sclerosis have a lower abundance of Verrucomicrobia and a higher abundance of Cyanobacteria. (Figure 2 A) This abundance of Cyanobacteria is particularly strange, because according to phyloseq, the Cyanobacteria in question all belong to the Chloroplast class. This raises the question of what chloroplasts were doing in the dark environment of the human intestine, and why Interferon apparently raised their abundance. This increased abundance is likely not an accident due to an outlier, because the standard deviation for Cyanobacteria abundance in Interferon patients is relatively similar to that of similarly abundant phyla. (Figure 2 B)

Although he phylogenetic tree of the overall microbiome data shows numerous clades but no correlation between disease status and phylogeny, it is too large to draw any detailed conclusions from. (Figure 3) The vast number of sequences in this microbiome data makes it impossible to determine any meaningful phylogenetic patterns at the entire-community level, because it contains so many different species. Therefore, I turned to smaller, phylum-specific analysis to draw more detailed conclusions.

##Firmicutes Community Analysis

The first phylum I looked at in-depth was Firmicutes, because it had one of the highest abundances in both the control group and all multiple sclerosis patients. (Table 1) Although there are many different species within this phylum present in the patients, there is no particular class, order, or family that is absent or present in one group of patients. (Table 3) This indicates that either the difference in Firmicutes microbiome content between multiple sclerosis and healthy patients is only apparent on the genus or species level, or there is no major difference for this phylum in multiple sclerosis patient's gut microbiomes. Based on previous studies, I suspect that it is the former, and the gut microbiome associated with multiple sclerosis within this class are only observable at smaller taxonomic levels. [@miyake2015dysbiosis]

In order to reach this conclusion, the next analysis I did was to find the Shannon diversity within the Firmicutes phylum. The analysis showed that the Shannon diversity within this phylum more or less mimicked the overall diversity for each patient type. (Figure 4) This makes sense, given that Firmicutes is the most common phylum and therefore likely accounts for most of the diversity in each patient type.

The next method I used to analyze the Firmicutes class was the phylogenetic tree of only bacterium in the Firmicutes phylum. While there are several distinct clades in this phylogegetic tree, all of them appear to contain samples from patients in each disease type. (Figure 5) Given that Firmicutes is one of the most abundant phyla in the human gut, these findings seem to indicate that the Firmicutes bacteria present in multiple sclerosis patients' guts play the same basic role as those in a healthy person's gut. [@turnbaugh2009core] That said,, it is possible that while a majority of Firmicutes play the same role in healthy controls and MS patients, there may be certain classes, or orders that are different within each patient type.

Therefore, my final analysis of the Firmicutes class was to look at the relative abundances of each Order within it. I cound that even at this level, the relative abundances were relatively similar across patient types, with a few variations. (Figure 10) Interestingly, the Lactobacillales appear to be less abundant than the control in the Copaxone treatment group, more abundant among the Interferon treatment group, and equally abundant among the Untreated group. Although it a strange occurrence, I think it has more to do with the drugs' side effects rather than the treatment of the disease, there is no difference in this order between the untreated and control groups. I think that this analysis indicates that any differences within the Firmicutes phylum are on a more specific taxonomic level than even Order.

##Verrucomicrobia Community Analysis

The next phylum I looked at was Verrucomicrobia, because it was very abundant in every patient group except for the Interferon-treated multiple sclerosis patients. (Figure 2 A) Just like Cyanobacteria, there is significantly lower Shannon diversity in Verrucomicrobia in patients of every sample, which makes sense because it is also less common. (Figure 8) However, unlike Cyanobacteria, there does not appear to be a significant phylogenetic difference between Verrucomicrobia in multiple sclerosis and healthy patients. (Figure 9) The lack of diversity in this phylum both between and within multiple sclerosis patient sand healthy individuals indicates that it is likely not important to the development or treatment of multiple sclerosis. That said, the analysis I performed did not explain why its abundance appears to be so much lower in Interferon-treated multiple sclerosis patients than the other groups.

The original paper by Jangi *et al.* also found that there was an increase in Verrucomicrobia in multiple sclerosis patients compared to healthy controls. [@jangi2016alterations] However, they did not divide the two treatment groups into two separate categories, and found that the average abundance between Copaxone and Interferon-treated multiple sclerosis patients is similar to the abundance in healthy patients. Although this is true, looking at the to treatments individually reveals that the average only appears this way because one group has much less Verrucomicrobia than the control, and one has slightly more. I think that my additional analysis indicates that the two treatment methods address different aspects of multiple sclerosis.

##Cyanobacteria Community Analysis

The last phylum I looked at in-depth was Cyanobacteria, both because it seemed to have different abundance patterns, and because it seemed like an odd bacterium to find inside a human gut. This struck me as peculiar, because Cyanobacteria is well-known as a photosynthetic phylum that needs light in order to complete its cellular processes. If this is the case then what would it be dong in the dark environment of the human intestine? Although this could be attributed to contamination or identification error, this seems unlikely because every patient type contains some amount of this phylum. (Figure 3 A) A more likely explanation could be that the DNA came from plant matter that the patients ate, which is supported by the fact that the Cyanobacteria sequences all appear to be from the Chloroplast class. (Table 4) However, this possibility does not explain why the Cyanobacteria form two distinct, somewhat well-supported clades that differ between multiple sclerosis patients and healthy controls. (Figure 7) Unless the multiple sclerosis patients were all eating the same plant material, and said plant material was different from that which the healthy controls were eating, this observation would mot make sense. Because of this, I think it is not unreasonable to infer that the Cyanobacteria were in fact living inside the patients' guts. Therefore, the following analysis assumes that the Cyanobacteria sequences *are* symbiotes living in the patients' digestive tracts, and not from plant matter they had consumed.

Initially, it appears that patients in the Interferon-treated multiple sclerosis group appeared to have a greater abundance of this phylum than the other groups. (Figure 2 A) That said, this could be due to an outlier within the Interferon group, as this group shows a greater standard deviation among Cyanobacteria than the other groups. (Figure 2 B) This fact observation being due to an outlier is further supported by the diversity metrics within this phylum, which shows one Interferon patient with significantly higher Shannon diversity than the other groups. (Figure 6) Interestingly, all of the Cyanobacteria appeared to be from the same class, Chloroplast. (Table 4) As one might expect from this information, the alpha diversity within this phylum was subsequently low among all individuals, including Interferon-treated multiple sclerosis patients. (Figure 6) Measuring diversity alone, it might appear that Cyanobacteria has little impact on the pathogenesis of multiple sclerosis. That said, these diversity measures only account for diversity within patients of the same disease status, and looking at the genetic trends within patient types reveals a different trend.

Phylogenetic analysis indicates that even if the abundance and diversity of Cyanobacteria is not significantly different within the same categories of  multiple sclerosis patients and healthy controls, this phylum still may play a role in the disorder. The phylogenetic analysis shows a more meaningful difference between the Cyanobacteria in healthy and multiple sclerosis patients. This tree showed two main clades, one that primarily contained Cyanobacteria from healthy patients and another that contained Cyanobacteria exclusively from multiple sclerosis patients. (Figure 7) This indicates that while there is little diversity in Cyanobacteria within patient groups, the Cyanobacteria found in multiple sclerosis patients appears to be genetically different than that in healthy patients. This indicates that there is a difference in the Cyanobacteria contents of gut microbiomes between multiple sclerosis and healthy patients, and possibly a connection between this phylum and the development or treatment of multiple sclerosis.

If there is Cyanobacteria plays a significant role in the development of multiple sclerosis, the next step in understanding this role would be to figure out the mechanism through which it does so. One study links multiple sclerosis to a phage that is known to infect certain species of Cyanobacteria. [@carter2012epstein] Given this information, I think it is possible that the Cyanobacteria in these multiple sclerosis patients may be somehow related to this virus. One possibility is that these strains are somehow more susceptible to the virus, causing multiple sclerosis patients' Cyanobacteria strains to become infected while healthy controls' Cyanobacteria to remain uninfected. Another possibility is what the Cyanobacteria in multiple sclerosis patients have incorporated viral DNA into their genome or as a plasmid, although this might not be evident in a 16S sample. I think it would be interesting to perform a more in-depth study on the Cyanobacteria in these patients' gut microbiomes, possibly looking for viral genetic markers or other, more Cyanobacteria-specific amplicons.

# Sources Cited
